jbpm-wb-integration/jbpm-wb-integration-backend/src/test/java/org/jbpm/workbench/wi/backend/server/dd/DDEditorServiceTest.java

```diff
@@ -262,6 +262,36 @@ public void testValidateMvelResolver() {
     }
 
     @Test
+    public void testValidateMvelResolverWithSupportedParameters() {
+        List<NamedObjectModel> models = Arrays.asList(new NamedObjectModel[]{
```

Comment:
`Arrays.asList()` has a varargs argument, so you don't actually need to create the wrapper array - just use the object directly.